### DISCUSSION TITLE
Overall test in `lincom`

### DISCUSSION THREAD

**Discussion by cwolock** [statdivlab/rigr#62] [General]
I think this is mostly a question for @bdwilliamson. 

How do we feel about the overall test in `lincom`? I think it's really useful to be able to give a matrix for `comb` in order to do multiple linear combinations in a single command, but I'm not sure about the overall test. It seems like it takes the sum of the given linear combinations and tests that? I wonder if this will be confusing to students. And is in terms of specifying a null hypothesis for the overall test, it looks like it's currently hard-coded to be 0. 

Not knowing anything about how people use this function, I would lean toward getting rid of the overall test. Thoughts?

**Comment by bdwilliamson** [comment]
I'm happy to remove the overall test -- your understanding of the interpretation seems correct, and I'm not sure how often it's used. If someone wants it later on, we can add it back in.

**Reply by cwolock** [reply]
Great, thanks for the input. It's been removed.
